pkg/sql/execinfrapb/processors.proto

```diff
@@ -101,7 +101,7 @@ message ProcessorCoreUnion {
   optional BackfillerSpec backfiller = 11;
   optional ReadImportDataSpec readImport = 13;
   reserved 14;
-  optional CSVWriterSpec CSVWriter = 20;
+  optional ExportSpec exporter = 20;
```

Comment:
@yuzefovich perhaps its just safer to push ExportSpec to a new processor number?